exporter/awss3exporter/config.go

```diff
@@ -32,6 +32,8 @@ type S3UploaderConfig struct {
 	S3ForcePathStyle bool `mapstructure:"s3_force_path_style"`
 	// DisableSLL forces communication to happen via HTTP instead of HTTPS.
 	DisableSSL bool `mapstructure:"disable_ssl"`
+
+	StorageClass string `mapstructure:"storage_class"`
```

Comment:
Does this need to be used anywhere?

Reply:
Thanks for the heads ups,  at some point during a rebase with main branch I lost the functionality